Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocAutoCompleteReferenceValue.css

```diff
@@ -1,248 +1,146 @@
-  top: 0;
-  left: 0;
-  height: 1.65em; /* for Drop Down Button */
-  white-space: nowrap; /* for Drop Down Button */
+  background-color: var(--color-standard-background);
 }
 
-span.bocAutoCompleteReferenceValue.readOnly span.content
+.bocAutoCompleteReferenceValueButton img
 {
-  position: static;
-  height: auto;
-}
-
-span.bocAutoCompleteReferenceValue.readOnly span.content span:empty
-{
-  /* Ensures that empty readonly values are not collapsed and can therefor visualize the keyboard focus */
-  min-width: 2em;
-  min-height: 1.5em;
-  vertical-align: -0.2em;
-}
-
-span.bocAutoCompleteReferenceValue.readOnly span.body
-{
-  /* Ensures that diacritic marks can be rendered without being cut off. 
-     Using line-height instead of padding works better with BocAutoCompleteReferenceValue. When using padding, the text is not aligned with the label. */
-  height: 1.4em;
-  /* Ensures that the outline is rendered outside of the space reserved for diacritic marks. */
-  display: inline-block;
-}
-
-span.bocAutoCompleteReferenceValue.readOnly span.content.hasOptionsMenu span,
-span.bocAutoCompleteReferenceValue.readOnly span.content.withoutOptionsMenu span
-{
-  /* Ensures that diacritic marks can be rendered without being cut off. 
-     Using line-height instead of padding works better with BocAutoCompleteReferenceValue. When using padding, the text is not aligned with the label. */
-  line-height: 1.4em;
-  /* Ensures that the outline is rendered outside of the space reserved for diacritic marks. */
-  display: inline-block;
-}
-
-/* has Icon */
-span.bocAutoCompleteReferenceValue span.content.hasIcon
-{
-  left: 22px; /* width of a.command icon + desired space between */
-}
-
-span.bocAutoCompleteReferenceValue span.icon img
-{
-  width: 16px;
-  height: 16px;
-  display: inline;
-}
-
-span.bocAutoCompleteReferenceValue span.optionsMenu
-{
-  position: absolute;
-  display: inline-block;
-  top: 0;
-  right: 0;
-  width: 6.7em; /* Define width of span.optionsMenu */
-}
-
-span.bocAutoCompleteReferenceValue span.content.hasOptionsMenu
-{
-  right: 7em; /* width of span.optionsMenu + desired space between */
-}
-
-span.content.hasOptionsMenu span.bocAutoCompleteReferenceValueInput,
-span.content.withoutOptionsMenu span.bocAutoCompleteReferenceValueInput
-{
-  display: inline-block;
-  width: auto;
-  position: absolute;
-  left: 0;
-  right: 24px;
-}
-
-.mozilla span.content.hasOptionsMenu span.bocAutoCompleteReferenceValueInput,
-.mozilla span.content.withoutOptionsMenu span.bocAutoCompleteReferenceValueInput
-{
-  right: 26px;
+  width: var(--remotion-themed-img-width);
+  height: 100%;
+  background-image: url(../Image/sprite.svg#AutoCompleteReferenceValueDropDownButtonFace);
+  background-position: center;
+  background-repeat: no-repeat;
 }
 
-span.bocAutoCompleteReferenceValueInput input
+.bocAutoCompleteReferenceValue span.bocAutoCompleteReferenceValueButton
 {
-  height: 1.2em;
-  line-height: 1em;
+  position: relative;
+  float: right;
```

Comment:
Is `float: right` really needed, or can this also be done with a flex property?

Reply:
Since the button actually overlays the input, I think it has to be done by `float: right` since flex does not support this behavior, at least that's the way I understood it.